lib/special_cases.php

```
@@ -399,3 +400,37 @@ function fgetcsv($stream, ?int $length = null, string $separator = ",", string $
     }
     return $safeResult;
 }
+
+/**
+ * The passthru function is similar to the
+ * exec function in that it executes a
+ * command. This function
+ * should be used in place of exec or
+ * system when the output from the Unix command
+ * is binary data which needs to be passed directly back to the
+ * browser.  A common use for this is to execute something like the
+ * pbmplus utilities that can output an image stream directly.  By
+ * setting the Content-type to image/gif and
+ * then calling a pbmplus program to output a gif, you can create
+ * PHP scripts that output images directly.
+ *
+ * @param string $command The command that will be executed.
+ * @param int|null $result_code If the result_code argument is present, the
+ * return status of the Unix command will be placed here.
+ * @throws ExecException
+ *
+ */
+function passthru(string $command, ?int &$result_code = null): void
+{
+    error_clear_last();
+
+    if (null === $result_code) {
+        $safeResult = \passthru($command);
+    } else {
```

IMHO, this will introduce a BC-break.
Indeed, the following commands will not have the same result:

```php
$code1 = null;
passthru('dosomething', $code1);
var_dump($code1); // 127

$code2 = null;
\Safe\passthru('whoami', $code2);
var_dump($code2); // null
```

---

Yea i see.

---

i reverted it and ignored the error